Ryan Bullough/week01/Thursday/main.js

```diff
@@ -0,0 +1,258 @@
+// ### Part 1, Rectangle
+
+// Given the following a `rectangle` object like the one below, write the following functions which accept a `rectangle` object as an argument:
+
+// * isSquare - Returns whether the rectangle is a square or not
+// * area - Returns the area of the rectangle
+// * perimeter - Returns the perimeter of the rectangle
+
+const rectangleA = {
+  length: 4,
+  width: 5,
+};
+
+const rectangulator = (rectangle) => {
+  const width = rectangle.width;
+  const length = rectangle.length;
+  const rectangleProperties = {
+    isSquare: width === length,
+    area: width * length,
+    perimeter: width * 2 + length * 2,
+  };
+  console.log(rectangleProperties.isSquare);
+  console.log(rectangleProperties.area);
+  console.log(rectangleProperties.perimeter);
+};
```

Comment:
Try to get into the habit of using return statements rather than console.logs(). This will help when we start doing more with JS and we want to do other things with our functions than just show the result on the console.